### PR TITLE
Feature/jng 3769 improve logging

### PR DESCRIPTION
**Pull request by robertcsakany**

### CODE REVIEW

File: epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/impl/LogLevel.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public enum LogLevel {
+    ERROR, WARN, INFO, DEBUG, TRACE;
+
+    public static Set<LogLevel> getMatchingLogLevels(LogLevel logLevel) {
```
Comment (noherczeg) [inline epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/impl/LogLevel.java:20]: Isn't this a bit heavy, creating a Set every time someone tries to query it?

File: epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/model/excel/ExcelColumn.java [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-	{
-		super(worksheet, index);
-	}
-}
+package hu.blackbelt.epsilon.runtime.execution.model.excel;
+
+import org.eclipse.epsilon.emc.spreadsheets.SpreadsheetColumn;
+import org.eclipse.epsilon.emc.spreadsheets.SpreadsheetWorksheet;
+
+public class ExcelColumn extends SpreadsheetColumn {
```
Comment (noherczeg) [inline epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/model/excel/ExcelColumn.java:6]: What is the actual change here?

File: epsilon-runtime-osgi/pom.xml [hunk 3]
```diff
@@ -16,6 +16,8 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.6</version>
```
Comment (noherczeg) [inline epsilon-runtime-osgi/pom.xml]: Shouldn't this version come from the parent pom?

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: epsilon-runtime-osgi/pom.xml (hunk 3 above)
Comment (robertcsakany) [reply epsilon-runtime-osgi/pom.xml]: Idea have no idea what is the version....

File: epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/model/excel/ExcelColumn.java (hunk 2 above)
Comment (robertcsakany) [reply epsilon-runtime-execution/src/main/java/hu/blackbelt/epsilon/runtime/execution/model/excel/ExcelColumn.java:6]: Nothing, namespace

**Pull request merged**